index.bs

```diff
@@ -5523,6 +5529,126 @@ opened</dfn> steps given |window|, |type|, |message|, and optional |default valu
 
 </div>
 
+## The emulation Module ## {#module-emulation}
+
+The <dfn export for=modules>emulation</dfn> module contains commands and events
```

Comment:
Won't this become quite a large module over time? Would it be better to keep (eg) the geolocation emulation in a `geolocation` module? This would also aid discoverability, as all the commands to do with geolocation would all be in the same place.

Reply:
Are we expecting a lot of geolocation-specific functionality? To me it seems more likely that pattern would lead to a lot of modules for each thing we want to override (geolocation / timezone / locale / etc.) all containing one function to override that thing. Of course it doesn't really matter either way, but considering them all kinds of environment emulation seems reasonable to me.